bevm/bevm_contract_test.go

```diff
@@ -195,11 +195,33 @@ func Test_Time(t *testing.T) {
 	timeTestInstance, err := bevmClient.Deploy(txParams.GasLimit, txParams.GasPrice, 0, a, contract)
 	require.Nil(t, err)
 
-	// Get current block time
-	expectedTime := big.NewInt(12345) // Currently hardcoded in getContext()
-	time, err := bevmClient.Call(a, timeTestInstance, "getTime")
+	// Get current timestamp in [s]
+	now := time.Now().Unix()
+
+	// Retrieve stored time
+	storedTime, err := bevmClient.Call(a, timeTestInstance, "getStoredTime")
+	require.Nil(t, err)
+	// Unitialized -- 0
+	assertBigInt0(t, storedTime.(*big.Int))
+
+	// Store current time
+	err = bevmClient.Transaction(txParams.GasLimit, txParams.GasPrice, 0, a,
+		timeTestInstance, "storeCurrentTime")
+	require.Nil(t, err)
```

Comment:
```suggestion
	require.NoError(t, err)
```
for consistency that would require to update all the other ones in this file. Could be done in another PR.

Reply:
in fact #2213 already updates all the other ones. So you should make sure the new ones actually use `require.NoError`

Reply:
Well spotted! :)
It was my intention to make the new ones correct, but I visibly missed some :unamused: .